cfgov/v1/forms.py

```diff
@@ -1,16 +1,49 @@
 from django import forms
 from django.forms.widgets import DateInput, SelectMultiple
+from sheerlike.templates import date_formatter
 from v1.models.events import EventPage
+from django.forms.utils import ErrorList
 
+class FilterErrorList(ErrorList):
+
+    def __str__(self):
+        return '\n'.join(str(e) for e in self)
+
+class FormDateField(forms.DateField):
+    def to_python(self, value):
+        if value:
+            try:
+                value = date_formatter(value)
+            except Exception as e:
+                raise forms.ValidationError(('"%s" is not a valid Date'), params=(value,))
+
+
+        return super(FormDateField, self).to_python(value)
 
 class CalenderPDFFilterForm(forms.Form):
-    filter_calendar = forms.CharField()
-    filter_range_date_gte = forms.DateField(input_formats=['%Y-%m-%d'])
-    filter_range_date_lte = forms.DateField(input_formats=['%Y-%m-%d'])
+    filter_calendar = forms.CharField(label='Calendars')
+    filter_range_date_gte = FormDateField(label='From', required=False)
+    filter_range_date_lte = FormDateField(label='To',required=False)
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/606b7c1d369eb210b18ded6cb7f660ffbe3bf87e